andela/firestar-backend
# #167730730 setup hound ci

#### What does this PR do?
This PR integrates Hound CI into the project
#### Description of Task to be completed?
Integrate Hound Ci into the repo
Create .hound.yml file to set rules
#### How should this be manually tested?
  - git clone this repo using the command `git clone git@github.com:andela/firestar-backend.git`
  - cd to the project
  - checkout to the branch using the command `git checkout ch-hound-setup-167730730`
  - Make changes or create a new branch, git add the change(s), commit and push to origin.
  - Create a pull request, scroll down to check for hound comments on code violation if there are any.
#### Any background context you want to provide?
Hound CI is a continuous integration software, that checks for code violations based on specified linting rules. Presently Hound CI neither generate badges nor expose stats for getting linting review result. Hence, the reason for the static the badge in the ReadMe file.
#### What are the relevant pivotal tracker stories?
[#167730730](https://www.pivotaltracker.com/story/show/167730730)
#### Screenshots (if appropriate)
![Screenshot (12)](https://user-images.githubusercontent.com/46751363/63263968-e03a5600-c281-11e9-8d10-64ac7fa59ec7.png)

#### Questions:
N/A
